docs/beyond_basics/101-147-riastores.rst

```diff
@@ -348,9 +354,9 @@ Afterwards, the dataset has two additional siblings: ``ria-backup``, and
 
    $ datalad siblings
 
-The storage sibling is the git-annex ora-remote and is set up automatically
-unless :command:`create-sibling-ria` is run with the ``--storage-sibling=off``
-flag. By default, it has the name of the RIA sibling, suffixed with ``-storage``,
+The storage sibling is the git-annex ora-remote and is set up automatically --
+unless :command:`create-sibling-ria` is run with ``--storage-sibling=off`` (in DataLad versions ``>0.14.``) or ``no-storage-sibling`` (in versions ``<0.14``).
```

Comment:
Missing `--` in front of `no-storage-sibling`?

Reply:
! yes, thanks :)